plugin/src/main/scala/com/geirsson/CiReleasePlugin.scala

```diff
@@ -89,7 +89,7 @@ object CiReleasePlugin extends AutoPlugin {
         } else {
           println("Tag push detected, publishing a stable release")
           sys.env.getOrElse("CI_RELEASE", "+publishSigned") ::
```

Comment:
Could this be necessary as well? I can't tell from the sbt-sonatype readme, but my intuition is that the speedups are gained from not running `publishSigned` (which is usually bottleneck).
```suggestion
          sys.env.getOrElse("CI_RELEASE", "+publishLocalSigned") ::
```

Reply:
My reading was that `publishSigned` goes into an archive thing locally if you set `publishTo := sonatypePublishToBundle.value` and then it's all shipped atomically when you do `sonatypeBundleRelease`. Maybe @xerial can clarify?

Reply:
Thank makes sense, let's give it a try and see it how fares!

Reply:
Looks like it failed with

```
[error] java.io.IOException: Supplied file /home/travis/build/olafurpg/sbt-ci-release/target/sonatype-staging/sbt-ci-release-1.3.0 is a not an existing directory!
```
The local directory was 

```
info] 	published sbt-ci-release to /home/travis/build/olafurpg/sbt-ci-release/target/sonatype-staging/plugin-1.3.0/com/geirsson/sbt-ci-release_2.12_1.0/1.3.0/sbt-ci-release-1.3.0-javadoc.jar.asc
```

It looks like `sonatypePublishToBundle` is using the project's `id` and `sonatypeBundleRelease` is using the `moduleName` 🤔

Reply:
Logs https://travis-ci.org/olafurpg/sbt-ci-release/jobs/581818663#L397

Reply:
:-\

Reply:
With publishTo := sonatypePublishToBundle.value setting, publishSigned task will create a local staging folder. 

It's a bit confusing, but publishLocalSigned task is for storing artifacts into ~/.ivy2/ folder.